docs/source/conf.py

```diff
@@ -40,7 +44,7 @@
     "sphinx.ext.napoleon",
     "sphinx.ext.mathjax",
     "sphinx.ext.autosummary",
-    "sphinx.ext.autosectionlabel",
+    # "sphinx.ext.autosectionlabel",
```

Comment:
Because of this:
```
Extension error (sphinx.ext.autosectionlabel):
Handler <function register_sections_as_label at 0x12cbeb6d0> for event 'doctree-read' threw an exception (exception: 'NoneType' object has no attribute 'parent')
make: *** [html] Error 2
```

There was no more traceback than that and it was only used once at the moment. I think it is incompatible with updated `sphinx`?

Reply:
It's fine, but probably merits a comment to say why you've done that.

Going forward, you probably want to check that documentation builds without warnings/errors using GitHub actions to avoid breakages like this.